src/Data/Nat/Properties.agda

```diff
@@ -1062,6 +1062,11 @@ m^n≡1⇒n≡0∨m≡1 m (suc n) eq = inj₂ (m*n≡1⇒m≡1 m (m ^ n) eq)
 m^n≢0 : ∀ m n .{{_ : NonZero m}} → NonZero (m ^ n)
 m^n≢0 m n = ≢-nonZero (≢-nonZero⁻¹ m ∘′ m^n≡0⇒m≡0 m n)
 
+2^r>0 : ∀ (r : ℕ) → 2 ^ r > 0
```

Comment:
As per the [style guide](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#variables-1), can we use the names `n`, `m` etc. for natural number variables? (applies to several proofs in this PR).

Reply:
Done!